cluster/local/kind.sh

```diff
@@ -77,24 +67,29 @@ case "${1:-}" in
     copy_image_to_cluster ${BUILD_IMAGE} "${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}" "${KIND_NAME}"
 
     [ "$2" ] && ns=$2 || ns="${DEFAULT_NAMESPACE}"
-    echo "installing helm package(s) into \"$ns\" namespace"
-    ${HELM} install --name ${PROJECT_NAME} --namespace ${ns} ${projectdir}/cluster/charts/${PROJECT_NAME} --set image.pullPolicy=Never,imagePullSecrets=''
+    echo "installing helm package into \"$ns\" namespace"
+    ${KUBECTL} get namespace ${ns} 2>/dev/null||${KUBECTL} create namespace ${ns}
```

Comment:
i had been working on this same issue this weekend :)

another option with helm3 is to use `--create-namespace`, from https://stackoverflow.com/a/60817927/467462

small world though, that stackoverflow post mentions the upstream helm issue on this, opened by none other than illya 🙇‍♂️ 
https://github.com/helm/helm/issues/6794